src/error_messages/information_messages.h

```diff
@@ -11,6 +11,9 @@
 #ifndef INFO_MESSAGES_H
 #define INFO_MESSAGES_H
 
+/* Vulnerability-Detector info messages*/
+#define VU_NO_OSINFO_AG       "(5700): Unable to get the OS information for agent '%.3d'. It may not have the OS inventory enabled."
```

Comment:
Regarding the issue comment, the two causes that may cause this log to be printed are:

- Syscollector (OS data at less) is disabled on that agent.
- The agent OS data table has not been synced to the manager yet.

The second one will be the cause in most cases so I think it would be better to mention it instead of the possibility of having the inventory disabled (which is enabled by default).

I suggest something like:
```
#define VU_NO_OSINFO_AG "(5700): Unable to get the OS information for agent '%.3d'. Inventory data may not yet be synchronized."
```